shenyu-client/shenyu-client-core/src/main/java/org/apache/shenyu/client/core/client/AbstractContextRefreshedEventListener.java

```diff
@@ -108,7 +112,24 @@ public void onApplicationEvent(@NonNull final ContextRefreshedEvent event) {
         }
         publisher.publishEvent(buildURIRegisterDTO(context, beans));
         beans.forEach(this::handle);
+        Map<String, Object> apiModules = context.getBeansWithAnnotation(ApiModule.class);
+        apiModules.forEach(this::handleApiDoc);
     }
+
+    private void handleApiDoc(final String name, final Object classes) {
```

Comment:
`Object classes` rename to `Object bean`.

Reply:
fixed